packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1008,8 +1008,9 @@ class ListView extends BoxScrollView {
   /// child that could possibly be displayed in the list view instead of just
   /// those children that are actually visible.
   ///
-  /// It is usually more efficient to create children on demand using
-  /// [ListView.builder].
+  /// The widget generated by this constructor always keeps using the list initially
+  /// given. Thus, if you want to apply changes of children dynamically, it is more
+  /// efficient to create them on demand using [ListView.builder].
```

Comment:
```suggestion
  /// It is usually more efficient to create children on demand using
  /// [ListView.builder] because it will create the widget children lazily as necessary.
```